packages/nodes-base/nodes/Google/Gmail/v2/utils/draft.ts

```diff
@@ -4,27 +4,43 @@ import type { IEmail } from '@utils/sendAndWait/interfaces';
 
 import { googleApiRequest } from '../../GenericFunctions';
 
+function setThreadHeaders(
+	email: IEmail,
+	thread: { messages: Array<{ payload: { headers: Array<{ name: string; value: string }> } }> },
+): void {
+	if (thread?.messages) {
+		const lastMessage = thread.messages.length - 1;
+		const messageId = thread.messages[lastMessage].payload.headers.find(
+			(header: { name: string; value: string }) => header.name === 'Message-ID',
+		)?.value;
+
+		email.inReplyTo = messageId;
+		email.reference = messageId;
```

Comment:
this should be references
```suggestion
		email.references = messageId;
```